### DISCUSSION TITLE
Automatic filling fields values with values ​​equal to the filter values

### DISCUSSION THREAD

**Discussion by Vladimir-Va** [nocodb/nocodb#398] [Features]
If we setup filters with Column_name is_equal Values it is logical that when adding a new row, we want to get the value in the columns equal to the values ​​set in the filters.
Add checkbox"Fill" in filter row.
This will save users time and add little bit automation!

**Comment by o1lab** [comment]
This is good - but it has to be indicated clearly that fields are populated because of filter.

**Reply by Vladimir-Va** [reply]
Example:
"Fill" checkbox visible if operation = is equal
![nocodb_filters](https://user-images.githubusercontent.com/52074398/127852215-7f45562a-b76c-4a42-add2-b643b67abf76.png)
